src/components/main/main.scss

```
@@ -1,3 +1,189 @@
+$primarycolor:#000;
+@import url('https://fonts.cdnfonts.com/css/helvetica-neue-9');
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:opsz,wght@6..12,200&display=swap');
 .espaco{
     height: 100vh;
 }
+.content-initial, .second-content, .third-content, .fourth-content{
+    display: flex;
+    width: 100%;
+    margin-bottom: 90px;
+    overflow-x: hidden;
+}
+.content-box, .third-content-box{
+    display: flex;
+    width: 100%;
+    padding: 0 260px 0 260px;
```

Aqui se você quiser, pode usar desta forma:
```suggestion
    padding: 0 260px;
```